3drepo/3drepo.io
# Issue 4867 - V4 manual group validation broken with hidden meshes

This fixes #4867 

#### Description
- New groups do not show "last upated" or "objects count"
- Selected nodes (in redux state) should always reflect what is effectively selected. As such, selecting all the objects and the hiding them all will make a new group invalid

#### Test cases
- Select all the objects;
- Click on "new group" (the group should be saveable)
- Hide all the objects; (the groups should not be saveable)
